ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -56,6 +56,9 @@ type WorkerGroupSpec struct {
 	// MaxReplicas denotes the maximum number of desired Pods for this worker group, and the default value is maxInt32.
 	// +kubebuilder:default:=2147483647
 	MaxReplicas *int32 `json:"maxReplicas"`
+	// NumOfHosts denotes the number of hosts to create per replica. The default value is 1.
+	// +kubebuilder:default:=1
+	NumOfHosts *int32 `json:"numOfHosts"`
```

Comment:
1. Should we add `omitempty` to make this field optional?
2. Perhaps we could use `int32` instead to avoid some nil checks.

Reply:
Fixed.